### PR TITLE
VDDK version matches VMware version

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIGENG-798

Also added supported VMware versions

CP for 4.5, 4.6

### CODE REVIEW

**Comment by istein1** [comment]
lgtm

File: modules/virt-creating-vddk-image.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ $ mkdir /tmp/<dir_name> && cd /tmp/<dir_name>
 
 . In a browser, navigate to link:https://code.vmware.com/home[VMware code] and click *SDKs*.
 . Under *Compute Virtualization*, click *Virtual Disk Development Kit (VDDK)*.
-. Select the latest VDDK release, click *Download*, and then save the VDDK archive in the temporary directory.
+. Select the VDDK version that corresponds to your VMware vSphere version, for example, VDDK 7.0 for vSphere 7.0, click *Download*, and then save the VDDK archive in the temporary directory.
```
Comment (aburdenthehand) [inline modules/virt-creating-vddk-image.adoc:25]: If "VDDK 7.0" is a selectable item in a list I think it should be marked up as
`*VDDK 7.0* `

Comment (apinnick) [reply modules/virt-creating-vddk-image.adoc:25]: No, it is not a selectable item in a list. Change request rejected.

**Pull request merged**